### PR TITLE
Philip/first version

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,jsx,ts,tsx}]
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+# Tab indentation (no size specified)
+[Makefile]
+indent_style = tab
```
Comment (musnit) [inline .editorconfig]: u prefer tab over spaces?

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (musnit) [inline public/manifest.json]: fix manifest names

File: src/interfaces/tendermint.ts [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  },
+  block_gossip_params: {
+    block_part_size_bytes: string,
+  },
+  evidence_params: {
+    max_age: string,
+  }
+}
+
+export interface IRpcGenesis {
```
Comment (musnit) [inline src/interfaces/tendermint.ts]: I haven't really used TypeScript much yet, so this is a question more than a suggestion, but does this imply the Genesis JSON structure is statically typed, and so will not work if the backend returns a different structure?
eg: as we add new stuff the the CLP module, there will be new fields in the genesis

Will this mean the client needs to have exact genesis json structure compatibility with the server, or not? If so, maybe we could be a bit more dynamic here and allow unexpected fields to come from the server?

File: .editorconfig (hunk 1 above)
Comment (philipstanislaus) [reply .editorconfig]: No, Makefiles seem to require tabs. All other files are using spaces.

File: public/manifest.json (hunk 2 above)
Comment (philipstanislaus) [reply public/manifest.json]: Good find, done!

File: src/interfaces/tendermint.ts (hunk 3 above)
Comment (philipstanislaus) [reply src/interfaces/tendermint.ts]: TypeScript does not change anything at runtime, it’s just helps development. It complains if you want to access a field that’s not defined in the interface, or if you have typos. That helps with autosuggestion and prevent errors. After compilation, all typescript code is removed, that means that it does not matter what JSON is coming from the server. 

 Hope that answers your question!

Comment (philipstanislaus) [reply src/interfaces/tendermint.ts]: TypeScript does not change anything at runtime, it’s just helps development. It complains if you want to access a field that’s not defined in the interface, or if you have typos. That helps with autosuggestion and prevent errors. After compilation, all typescript code is removed, that means that it does not matter what JSON is coming from the server. 

 Hope that answers your question!

**Pull request merged**